docs/index.rst

```diff
@@ -132,9 +132,11 @@ Using ``pyvo``
 
    dal/index
    registry/index
+   discover/index
    io/index
    auth/index
    samp
    mivot/index
    utils/index
    utils/prototypes
+   utils/testing
```

Comment:
there is no `docs/utils/testing.rst` file, so this causes an error.

Reply:
Sorry about this; there *was* a testing.rst (about building scaffolds by observing HTTP requests, which fails because of version strings and such) and I forgot to remove that reference when I dropped it for this PR (I'll re-introduce testing helpers in another PR, though, but that's different material).